kthresher.bash_completion

```diff
@@ -15,31 +19,30 @@ genOpts()
   local IFS
   local not_used=""
   # options in pairs
-  local opts="--help,-h --config,-c --dry-run,-d --headers,-H --keep,-k --purge,-p --show-autoremoval,-s --verbose,-v --version,-V"
+  local opts="--help,-h --config,-c --dry-run,-d --headers,-H --keep,-k --purge,-p "
+  opts+="--show-autoremoval,-s --verbose,-v --version,-V"
   for i in ${opts} ; do
-    OLDIFS=$IFS
+    OLDIFS=${IFS}
     # set IFS to split our pairs
     IFS=","
-    set -- $i
+    set -- ${i}``
     # reset IFS to original value
-    IFS=$OLDIFS
-    result_one=$(containsElement "${1}" "${COMP_WORDS[@]}" ; echo $?)
-    result_two=$(containsElement "${2}" "${COMP_WORDS[@]}" ; echo $?)
-    # if neither exists in COMP_WORDS, both short and long opt are valid for use
-    if [[ "${result_one}" != 0 ]] && [[ "${result_two}" != 0 ]]; then
-      not_used="${not_used} ${1}"
-      not_used="${not_used} ${2}"
+    IFS=${OLDIFS}
+    if ! $( containsElement "${1}" "${COMP_WORDS[@]}" ) && \
+       ! $( containsElement "${2}" "${COMP_WORDS[@]}" ); then
+         not_used="${not_used} ${1}"
+         not_used="${not_used} ${2}"
```

Comment:
I've removed the extraneous `\` - I also tried moving the `&&` to the next line and leaving the `\` back in as I feel that is probably more logical to the brain but the former looked easier to read so ultimately went with that suggestion.

I don't see a visible difference in functionality between append the new values in one line vs in two, I'm made that adjustment as well. PR update incoming.